cult/theme.json

```diff
@@ -61,36 +61,90 @@
 			"customFontSize": true,
 			"fontFamilies": [
 				{
-					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+					"fontFamily": "Petrona, serif",
 					"name": "Body (System Font)",
-					"slug": "body-font"
+					"slug": "body-font",
+					"fontFace": [
+						{
+							"fontDisplay": "block",
+							"fontFamily": "Petrona",
+							"fontWeight": "300 700",
+							"fontStyle": "normal",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/Petrona.ttf"
+							]
+						},
+						{
+							"fontDisplay": "block",
+							"fontFamily": "Petrona",
+							"fontWeight": "300 700",
+							"fontStyle": "italic",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/Petrona-Italic.ttf"
+							]
+						}
+					]
 				},
 				{
-					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+					"fontFamily": "'Work Sans', sans-serif",
 					"name": "Headings (System Font)",
-					"slug": "heading-font"
+					"slug": "heading-font",
+					"fontFace": [
+						{
+							"fontDisplay": "block",
+							"fontFamily": "'Work Sans'",
+							"fontWeight": "900",
+							"fontStyle": "normal",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/WorkSans.ttf"
+							]
+						},
+						{
+							"fontDisplay": "block",
+							"fontFamily": "'Work Sans'",
+							"fontWeight": "900",
+							"fontStyle": "italic",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/WorkSans-Italic.ttf"
+							]
+						}
+					]
 				}
 			],
 			"fontSizes": [
+				{
```

Comment:
It looks like we are pretty inconsistent in the number of font sizes and their slugs:

[Archeo](https://github.com/Automattic/themes/blob/trunk/archeo/theme.json#L127-L153)
[Livro](https://github.com/Automattic/themes/blob/trunk/livro/theme.json#L104-L125)
[Pendant](https://github.com/Automattic/themes/blob/trunk/pendant/theme.json#L160-L186)

Should we try to tighten that up or is it inconsequential? I guess it has an impact if a user switches themes.

Reply:
Indeed. I had the same thought when adding "Extra Small" to Pendant and we've definitely discussed previously.

It feels wrong to limit designs to the core sizes.  It's definitely not enough.

It feels cumbersome to leverage sizes outside of defined sizes.  

The '[custom](https://github.com/Automattic/themes/blob/trunk/blockbase/theme.json#L175)' font sizes in Blockbase do this so we can use a variable in the theme.json or css but then not allow a user to select it.  We can't leverage those sizes in patterns though 'cause it's the same problem.  But that feels icky too.

Providing users with font sizes that might go away if they switch themes is ichy.

At a minimum we need to consolidate on our naming outside of the norm.  Is it "tiny" or "extra small".  At least then switching amongst our themes will yield the same results.

At a maximum we need to remove any font size outside of what's provided by core and just use hard-coded sizes instead of those variables.

Reply:
> At a maximum we need to remove any font size outside of what's provided by core and just use hard-coded sizes instead of those variables.

I think I stand here, since it keeps things more consistent, portable, and predictable. It provides for a nicer UI as well: 

Core supplied | Extra presets
--- | ---
<img width="269" alt="Screen Shot 2022-04-27 at 10 50 53 AM" src="https://user-images.githubusercontent.com/5375500/165558110-7268761b-0b45-44f6-ba22-0b6d18475c8f.png"> | <img width="270" alt="Screen Shot 2022-04-27 at 10 51 13 AM" src="https://user-images.githubusercontent.com/5375500/165558195-e081301e-4791-46df-a164-c1ed317fb3db.png">